packages/js/package.json

```diff
@@ -31,13 +31,13 @@
   "private": true,
   "scripts": {
     "clean": "rimraf ./dist",
-    "start": "pnpm run build -- --watch --sourcemap",
-    "pre:build": "cp ./src/ui/index.css ./src/ui/index.directcss",
-    "build": "pnpm run clean && pnpm run pre:build && tsup && pnpm run post:build",
+    "start:server": "http-server ./dist -p 4010",
```

Comment:
http-server will serve the `index.css` file with a URL like `http://localhost:4010/index.css`